doc: Add Generic Node Sensor Edition (#1037)
TheThingsIndustries/lorawan-stack-docs: doc/content/devices/generic-node-se/_index.md

---
title: "Generic Node Sensor Edition"
description: ""
weight: 
---

{{< figure src="generic-node-se.png" alt="Generic Node Sensor Edition" class="float plain" >}}

> This section will provide guidance on how to begin using [Generic Node Sensor Edition](https://www.genericnode.com/docs/sensor-edition/) on {{% tts %}}.

Author: Will do.

> The Generic Node Sensor Edition comes with a pre-loaded application called [vanilla](https://www.genericnode.com/docs/applications/se-vanilla/) that showcases the device's functions and capabilities. This application utilizes a combination of the onboard components including the LEDs, button, buzzer, accelerometer, secure element, temperature sensor, and humidity sensor. The instructions on how to flash a new application onto the Generic Node SE can be found [here](https://www.genericnode.com/docs/getting-started/se-sw/env-setup/stm32cubeide/).

Author: Yes, indeed.

# Onboarding Using the QR Code

The Generic Node can be onboarded to {{% tts %}} using the QR code which is printed on the back of its enclosure.

> - Sign in to [{{% tts %}}](/getting-started/).

Reviewer: Also, remove `-` bullets from this and following steps.

- In {{% tts %}} **Console**, choose the [application](/integrations/adding-applications/) to which you want to add the Generic Node.
- Select the **Register end device** button.

{{< figure src="application-overview.png" alt="Application overview" >}}

- In the **Register end device** page select the **Scan end device QR code** button.

{{< figure src="register-end-device.png" alt="Register end device" >}}

> - The camera will start up and be ready to scan the QR code. To begin, bring the QR code for your device close to the camera.

Reviewer: ```suggestion
- The camera will start up and be ready to scan the QR code. To scan the device, bring the QR code close to the camera.
```

{{< figure src="waiting-for-camera.png" alt="Waiting for camera" >}}

> - A window with information from the QR code should appear on your screen, displaying the **Claim authentication code**, **JoinEUI**, **DevEUI**, and **Brand** for your device. If the data displayed is correct, select the **Apply** button.

Reviewer: ```suggestion
- A window with information scanned from the QR code (like **Claim authentication code**, **JoinEUI**, **DevEUI** and **Brand**) should appear on your screen. If the data displayed is correct, select the **Apply** button.
```

{{< figure src="found-qr-data.png" alt="Found QR data" >}}

To continue with the rest of the registration process you can choose either **LoRaWAN Device Repository** or **manual registration**.

> ## Using the LoRaWAN Device Repository

Reviewer: ```suggestion
### Using the LoRaWAN Device Repository
```

> The [Device repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Generic Node Sensor Edition, which requires less information for registration with {{% tts %}}.

Reviewer: ```suggestion
The [LoRaWAN Device Repository](https://github.com/TheThingsNetwork/lorawan-devices) comprises over 600 end-device profiles, including the Generic Node Sensor Edition, which requires less information for registration with {{% tts %}}.
```

> - In the **End device type** section, under **Input Method** select **Select the end device in the LoRaWAN device repository**. Then select/enter the following settings:

Reviewer: Remove the bullet

  - **End device brand**: The Things Industries
  - **Model**: Generic Node (Sensor Edition)
  - **Hardware Ver.**: 1.1
  - **Firmware Ver.**: 1.1
  - **Profile (Region)**: choose EU_863_870 for Europe or US_902_928 for US

>   - **Frequency plan**: choose the frequency plan appropriate for your region/country, for example, if you are live in Europe, choose **Europe 863-870 MHz (SF9 for RX2 - recommended)**.

Reviewer: For selected values use **bold** and for entered values use `highlight`

{{< figure src="dev-rep-1.png" alt="Settings for registration through device repository" >}}

  - **JoinEUI**: pre-filled
  - **DevEUI**: pre-filled (this is the secure element’s DevEUI)
  - **AppKey**: select the **Generate** button.
  - **End device ID**: you can continue with the pre-filled value or use any other unique value.

> - Select the **Register end device** button.

Reviewer: Remove bullet

{{< figure src="dev-rep-2.png" alt="Settings for registration through device repository" >}}

> The device is now registered and you will be redirected to the **overview** page of the device.

Reviewer: ```suggestion
The device is now registered and you will be redirected to the **Overview** page of the device.
```

{{< figure src="gnse-after-register.png" alt="GNSE overview page" >}}

> ## Manually

Reviewer: ```suggestion
### Manual Registration
```

- In the **End device type** section, under **Input Method** select **Enter end device specifics manually**.
 Then select the following settings:
  - **Frequency plan**: choose the frequency plan appropriate for your region/country, for example, if you are live in Europe, choose **Europe 863-870 MHz (SF9 for RX2 - recommended)**.
  - **LoRaWAN version**: LoRaWAN Specification 1.0.2
  - **Regional parameters version**: RP001 Regional Parameters 1.0.2 revision B
- Select **Show advanced activation, LoRaWAN class and cluster settings** to expand the section. Ensure that the default selection is **Over the Air Activation (OTAA)**.

{{< figure src="manually-1.png" alt="Settings for manual registartion" >}}

- Under **Provisioning information** enter the following settings:
  - **JoinEUI**: pre-filled
  - **DevEUI**: pre-filled (this is the secure element’s DevEUI)
  - **AppKey**: select the **Generate** button.
  - **End device ID**: you can continue with the pre-filled value or use any other unique value.
- Select the **Register end device** button.

{{< figure src="manually-2.png" alt="Settings for manual registartion" >}}

> The device is now registered and you will be redirected to the **overview** page of the device.

Reviewer: Same comments as for the section above
